cmd/access/node_builder/access_node_builder.go

```diff
@@ -1673,6 +1673,7 @@ func (builder *FlowAccessNodeBuilder) Build() (cmd.Node, error) {
 				builder.unsecureGrpcServer,
 				builder.stateStreamBackend,
 				builder.stateStreamConf,
+				builder.ExecutionIndexer,
```

Comment:
I don't think this is going to work as expected. `builder.ExecutionIndexer` is instantiated after loading the db, which happens in parallel with startup for other components. It is likely going to be `nil` when the rpc engine is started, so it will never be set.

You will probably need to pass one of the proxy objects or create one for this usecase:
https://github.com/The-K-R-O-K/flow-go/blob/8c8014b64a17d40b244d02e00de29e6a07bd7b64/cmd/access/node_builder/access_node_builder.go#L838-L856